robot_log_visualizer/ui/gui.py

```diff
@@ -145,6 +158,7 @@ def __init__(self, signal_provider, meshcat_provider, animation_period):
 
         self.ui.actionQuit.setIcon(get_icon("close-circle-outline.svg"))
         self.ui.actionQuit.setIcon(get_icon("close-circle-outline.svg"))
+        self.ui.actionConnect.setIcon(get_icon("connection-outline.png"))
```

Comment:
can you use a svg as the other icons? Moreover I don't think the file is pushed

Reply:
Fixed in [f5b4a65](https://github.com/ami-iit/robot-log-visualizer/pull/80/commits/f5b4a6599825601b2c7a9bf35140fe0e9d979106). This is old code, a icon was never generated for the RT connect menu item. However, one can be made for the next PR